Workbooks/WorkItems/GitHub/GitHub.workbook

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": true,
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "5eaa3561-5587-44b0-b4d3-c372613fdf41",
+            "version": "KqlParameterItem/1.0",
+            "name": "operationid",
+            "type": 1,
+            "value": "",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "092e592b-96d2-44ae-825a-20c82c711818",
+            "version": "KqlParameterItem/1.0",
+            "name": "timestamp",
+            "type": 1,
+            "value": "",
+            "isHiddenWhenLocked": true
+          },
+          {
+            "id": "4d3ed30a-caa8-4a30-b518-904685e8a511",
+            "version": "KqlParameterItem/1.0",
+            "name": "httpendpoint",
+            "type": 1,
+            "value": "",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "paramValidationRules": [
+                {
+                  "regExp": "https:\\/\\/github\\.com\\/[^\\/]+\\/[^\\/]+$",
+                  "match": true,
+                  "message": ""
+                }
+              ]
+            }
+          },
+          {
+            "id": "9e031a62-faf3-4943-867a-a8e306da0ea5",
+            "version": "KqlParameterItem/1.0",
+            "name": "eventid",
+            "type": 1,
+            "isHiddenWhenLocked": true
```

Comment:
Just curious- what does this parameter mean?

Reply:
it will pass event id from detailsv2